compiler/src/securityAnnotations.ml

```diff
@@ -0,0 +1,113 @@
+open Utils
+open Prog
+
+(* Open questions / bikeshedding
+
+   - Is a pair (p, q) fine or should we use the more verbose { n:p, s:q }?
+*)
```

Comment:
We should sort out the question raised in this comment before merging, I guess.

Reply:
Personally, I prefer the more verbose and explicite version. Furthermore, since 
public, transient and secret are defined, I think most of the time the pair version will not be used.

Reply:
Do you have a concrete syntax to suggest (in particular, I find the `n` and `s` labels rather cryptic)?

Reply:
Done!